src/D3MHub.sol

```diff
@@ -255,7 +255,9 @@ contract D3MHub {
         if (mode == Mode.NORMAL) {
             // Normal mode or module just caged (no culled)
             // debt is obtained from CDP art
-            (,daiDebt) = vat.urns(ilk, address(_pool));
+            uint256 ink;
+            (ink, daiDebt) = vat.urns(ilk, address(_pool));
+            require(ink == daiDebt, "D3MHub/position-needs-to-be-fixed");
```

Comment:
I think this can be removed.
If it was left as an internal sanity check then it's probably redundant as we call `_fix()` before the two places where `Mode.NORMAL` is passed to unwind.

Reply:
Yeah totally, I had forgotten to remove it, it's done now.